CONTRIBUTING.md

```diff
@@ -125,7 +125,7 @@ CoC](https://github.com/rust-lang/rust/wiki/Note-development-policy#conduct).
 * Private harassment is also unacceptable. No matter who you are, if
   you feel you have been or are being harassed or made uncomfortable
   by a community member, please contact one of the channel ops or any
-  of the TC members immediately with a capture (log, photo, email) of
+  of the CTC members immediately with a capture (log, photo, email) of
```

Comment:
It looks like the text was copied from the nodejs Code of Conduct, which has this as the TSC, not the CTC, so yeah, this one should almost certainly be TSC and not CTC. See third-from-last bullet point at https://github.com/nodejs/node/blob/5fc6938cff48f978642d7d83e257de36660977d3/CODE_OF_CONDUCT.md#conduct